src/Bridge/Doctrine/EventListener/PublishMercureUpdatesListener.php

```diff
@@ -26,6 +26,7 @@
 use Doctrine\Common\EventArgs;
 use Doctrine\ODM\MongoDB\Event\OnFlushEventArgs as MongoDbOdmOnFlushEventArgs;
 use Doctrine\ORM\Event\OnFlushEventArgs as OrmOnFlushEventArgs;
+use Symfony\Bundle\MercureBundle\Mercure;
```

Comment:
This bridge must be usable without the Symfony framework. It can depend on the Mercure Component, but not on MercureBundle.
Maybe should we move this class in the component?